packages/suite/src/components/suite/FiatValue/index.tsx

```diff
@@ -34,23 +66,27 @@ const FiatValue = ({
     useCustomSource,
     showApproximationIndicator,
     disableHiddenPlaceholder,
-    ...props
 }: Props) => {
-    const targetCurrency = fiatCurrency ?? props.settings.localCurrency;
-    const currentFiatRates = props.fiat.coins.find(
+    const { fiat, settings } = useSelector(state => ({
+        fiat: state.wallet.fiat,
+        settings: state.wallet.settings,
+    }));
+
+    const targetCurrency = fiatCurrency ?? settings.localCurrency;
+    const currentFiatRates = fiat.coins.find(
         f =>
             f.symbol.toLowerCase() === symbol.toLowerCase() &&
             f.tokenAddress?.toLowerCase() === tokenAddress?.toLowerCase(),
     )?.current;
 
     const ratesSource = useCustomSource ? source : currentFiatRates?.rates;
-    const fiat = ratesSource ? toFiatCurrency(amount, targetCurrency, ratesSource) : null;
+    const currency = ratesSource ? toFiatCurrency(amount, targetCurrency, ratesSource) : null;
```

Comment:
it is not a currency, it stores fiat value of amount in given currency. fiat was slightly less confusing since there are variables such as targetCurrency that is passed to `currency` prop few lines below :)

Reply:
what about fiatValue/fiatAmount? :)

Reply:
`fiat` and `fiatCurrency` were already taken, couldn't think of anything better :smile:

Reply:
b5ba4db